### PR TITLE
253 display days in duration

### PR DESCRIPTION
**Pull request by DanielSeehausen**
image

(process would normally be blue and running)

### CODE REVIEW

File: src/components/misc/DurationDisplay.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+import {
+  getDisplayableDaysHoursMinutes
+} from "utils/time"
+
+
+const bigTimeStyle = {
```
Comment (DanielSeehausen) [inline src/components/misc/DurationDisplay.jsx]: chance for us to refactor this to jss with Mo when we do the duration display for the process tiles!

File: src/components/misc/DurationDisplay.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+const littleTimeStyle = {
+  fontSize: '18px',
+  color: '#bbb'
+}
+
+const littleDigit = {
+  fontSize: '15px',
+  margin: '0 2px 0 0'
+}
```
Comment (DanielSeehausen) [inline src/components/misc/DurationDisplay.jsx]: chance for us to refactor this styling to jss with Mo when we do the duration display for the process tiles!

File: src/components/misc/DurationDisplay.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    { `${hours}` }<span style={littleDigit}>h</span>{ `${minutes}` }<span style={littleDigit}>m</span>
+  </span>
+)
+
+
+const DurationDisplay = ({ ms }) => {
+  const { days, hours, minutes } = getDisplayableDaysHoursMinutes(ms)
+
+  return (
+    <div className='center aligned' style={bigTimeStyle}>
```
Comment (O-Susanoo) [inline src/components/misc/DurationDisplay.jsx]: class names probably from semantic ui ?

File: src/utils/time.js [hunk 4]
```diff
@@ -14,9 +14,12 @@ export const getDisplayDay = (date) => (
 export const getDisplayTime = (date) => (
   `${forceDoubleDigits(date.getHours())}:${forceDoubleDigits(date.getMinutes())}` // 14:08
 )
-export const getDisplayDuration = (ms) => (
-  `${forceDoubleDigits(ms/1000/60/60 | 0)}:${forceDoubleDigits((ms/1000/60 | 0)%60)}` // 01:04
-)
+
+export const getDisplayableDaysHoursMinutes = (ms) => ({
```
Comment (O-Susanoo) [inline src/utils/time.js]: Learning purpose

File: src/components/misc/DurationDisplay.jsx (hunk 3 above)
Comment (DanielSeehausen) [reply src/components/misc/DurationDisplay.jsx]: fixed! they were useless. good 👁

**Pull request merged**